### DISCUSSION TITLE
How do I go to a newly spawned tab on startup?

### DISCUSSION THREAD

**Discussion by sophisticatedduck** [wezterm/wezterm#4956] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I am using the following to spawn five tabs on startup:

```
wezterm.on('gui-startup', function()
  local _, first_pane, window = wezterm.mux.spawn_window {}
  local _, second_pane, _ = window:spawn_tab {}
  local _, third_pane, _ = window:spawn_tab {}
  local _, fourth_pane, _ = window:spawn_tab {}
  local _, fifth_pane, _ = window:spawn_tab {}
end)
```

This works great, however it starts off on the fifth tab on startup. I'd rather start on the first tab or first_pane when it finishes spawning the other tabs at startup. I tried adding `window:perform_action(act.ActivateTab(0), first_pane)` but that doesn't seem to work.

Any ideas would be appreciated. Thank you.

**Comment by wez** [comment, marked as answer]
```lua
wezterm.on('gui-startup', function()
  local first_tab, first_pane, window = wezterm.mux.spawn_window {}

  ...

  first_tab:activate()
end)
```

**Reply by sophisticatedduck** [reply]
Thanks! Worked perfectly.
